randovania/games/prime1/exporter/scan_tips.py

```diff
@@ -0,0 +1,696 @@
+            "\n\none can move through the room along a specific path while avoiding awakening the ghosts.\n"
+            "This can either be used to activate the Bomb Slots or to reach Crossway Access South."
+        ),
+    ],
+    "213585783": [
+        # Main Quarry Bombless Tip
+        # Replaces Main Quarry left computer scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;The nearby item is acquirable without Morph Ball Bombs by unmorphing on the crane "
+            "platform's edge.\nAlternatively, one can slope jump off of this "
+            "cabin's roof to scale the crane platform and reach the item."
+        ),
+    ],
+    "130269422": [
+        # Main Quarry Disposal Dash Tip
+        # Replaces Main Quarry right computer scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;It is possible to reach the door to Waste Disposal with a scan dash "
+            "and &main-color=#FF3333;Space Jump Boots&main-color=#89D6FF;."
+        ),
+    ],
+    "3474604570": [
+        # Control Tower Fight Skip Tip
+        # Replaces Control Tower ice scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;The Space Pirates in this room are alerted when one steps in front of the entrance "
+            "to West Tower. One can avoid this by jumping around the space directly in front of the entrance."
+        ),
+    ],
+    "3573990646": [
+        # Thardus Skip Tip
+        # Replaces Quarantine Monitor left screen scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;One can cross Quarantine Cave without fighting "
+            "Thardus by using aerial movement around the room's edges."
+        ),
+    ],
+    "3820360882": [
+        # Sun Tower Early Wild Tip
+        # Replaces Sun Tower gate scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;When entering from Sun Tower Access, one can awaken the Chozo Ghosts in Sunchamber "
+            "without falling to the bottom of the room. This requires either using the &main-color=#FF3333;Spider Ball"
+            "&main-color=#89D6FF; to roll to the bottom of the magnetic track before rolling back up, or stopping the "
+            "descent with an Oculus and using complex bomb jumps to rise back to the top."
+        ),
+    ],
+    "1530945290": [
+        # Training Chamber Ghost Skip Tip
+        # Replaces Meditation Fountain left sculpture scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;In Training Chamber, one can avoid awakening the Chozo Ghosts "
+            "by climbing architecture near the entrance to Training Chamber Access."
```

Comment:
This scan seems like too much. The trick in question is hypermode and the scan node is quite out of the way and inconspicuous. I think this should be removed.

Reply:
This is part of my first goal, mentioning every single Knowledge trick. The scan node is in a room with several other scan nodes containing tips - if you find one, you'll probably read them all. I think it would be reasonable to note that doing so is highly difficult, though.